### PR TITLE
feat(packages): Add harbor registry

### PR DESCRIPTION
**Pull request by EyeCantCU**
#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

File: harbor-registry.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  pipeline:
+    - runs: |
+        # The registry should start and begin listening before it's killed
+        registry_DO_NOT_USE_GC serve /etc/registry/config.yml & sleep 5; kill $!
+
+update:
+  enabled: true
+  github:
+    identifier: distribution/distribution
+    strip-prefix: v
```
Comment (ajayk) [inline harbor-registry.yaml:56]: Can you please add a basic test

File: harbor-registry.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      ln -sf /usr/bin/harbor-registry ${{targets.destdir}}/usr/bin/registry_DO_NOT_USE_GC
+      # Use example config as registry config
+      cp ./cmd/registry/config-example.yml ${{targets.destdir}}/etc/registry/config.yml
+
+test:
+  pipeline:
+    - runs: |
+        # The registry should start and begin listening before it's killed
+        registry_DO_NOT_USE_GC serve /etc/registry/config.yml & sleep 5; kill $!
+
```
Comment (ajayk) [inline harbor-registry.yaml:51]: you can enhance the test by hitting the endpoint and expecting a 200 OK response back on 
localhost:5000

File: harbor-registry.yaml (hunk 1 above)
Comment (EyeCantCU) [reply harbor-registry.yaml:57]: Added :)

File: harbor-registry.yaml (hunk 2 above)
Comment (EyeCantCU) [reply harbor-registry.yaml:52]: Thanks for the suggestion. Will get that up

Comment (EyeCantCU) [reply harbor-registry.yaml:57]: Added

**Review by ajayk — APPROVED** [review]

**Pull request merged**